setup.py

```diff
@@ -133,7 +133,14 @@ def gen_packages_items():
               'lmdeploy': lmdeploy_package_data,
           },
           include_package_data=True,
-          install_requires=parse_requirements('requirements.txt'),
+          setup_requires=parse_requirements('requirements/build.txt'),
+          tests_require=parse_requirements('requirements/test.txt'),
+          install_requires=parse_requirements('requirements/runtime.txt'),
+          extras_require={
+              'all': parse_requirements('requirements.txt'),
```

Comment:
So, it is one of the following, right?
- pip install lmdeploy[all]
- pip install lmdeploy[lite]
- pip install lmdeploy[serve]

Reply:
Can we use `pip install lmdeploy`?

Reply:
yes. we can install like:
- `pip install lmdeploy` or `pip install -e .`, which runs `pip install -r requirements/runtime.txt`
- `pip install lmdeploy[all]` or `pip install -e .[all]`, which runs `pip install -r requirements/runtime.txt -r requirements/lite.txt -r requirements/serve.txt`
- `pip install lmdeploy[lite]` or `pip install -e .[lite]`, which runs  `pip install -r requirements/runtime.txt -r requirements/lite.txt`
- `pip install lmdeploy[serve]` or `pip install -e .[serve]`, which runs  `pip install -r requirements/runtime.txt -r requirements/serve.txt`